static/js/game.js

```diff
@@ -186,10 +191,19 @@ function GameController(canvas) {
                 this.units[id]['rotation'] = unitData[key]
             }
         }
+        this.units[id].regX = this.units[id].width / 2;
+        this.units[id].regY = this.units[id].height / 2;
         this.units[id].speedTick = this.units[id].speed / this.frequency / FPS;
         this.update_shown_property();
     };
 
+    this.deleteUnit = function (unitData) {
```

Comment:
Let's name this method killUnit() - in future it may include animation of unit diying